### PR TITLE
Matti/disable work items feature

### PR DESCRIPTION
**Pull request by mmokko**
This contains a commit to add functionality for disabling a view and a commit to disable the work items feature.

Can they be merged without squashing or should they be separate PRs?

### CODE REVIEW

File: robocorp-code/vscode-client/src/extension.ts [hunk 1]
```diff
@@ -483,7 +483,7 @@ export async function activate(context: ExtensionContext) {
             roboCommands.ROBOCORP_RENAME_RESOURCE_IN_ROBOT_CONTENT_VIEW,
             renameResourceInRobotContentTree
         );
-        commands.registerCommand(roboCommands.ROBOCORP_UPDATE_LAUNCH_ENV, updateLaunchEnvironment);
```
Comment (fabioz) [inline robocorp-code/vscode-client/src/extension.ts:486]: The idea would be commenting this line out (the `updateLaunchEnv` should return the new env, so, if it returns something empty the implementation may be broken -- but if the command is not there the callers should be protected for that situation already).

**Review by fabioz — CHANGES_REQUESTED** [review]
Things can be merged without squashing if you'd rather work that way -- although you need some back and forth locally to fix the commits individually based on the review then...

File: robocorp-code/package.json [hunk 2]
```diff
@@ -657,9 +638,7 @@
         "compile": "cd vscode-client && tsc -p ./ && cd ..",
         "watch": "cd vscode-client && tsc -watch -p ./ && cd ..",
         "pretest": "cd vscode-client && tsc -p ./ && cd ..",
-        "test": "node ./vscode-client/out/tests/runTests.js",
-        "prettier": "npx prettier -c vscode-client/**/*.ts",
-        "prettier-fix": "npx prettier -w vscode-client/**/*.ts"
+        "test": "node ./vscode-client/out/tests/runTests.js"
```
[the remark this answers is not in the document]
Comment (mmokko) [reply robocorp-code/package.json:641]: I'll rebase from master.

File: robocorp-code/codegen/views.py [hunk 3]
```diff
@@ -239,13 +241,16 @@ def get_views_containers():
...
 
-def get_tree_views():
+def get_tree_views_for_package_json():
     ret = {}
 
     for tree_view_container in TREE_VIEW_CONTAINERS:
         ret[tree_view_container.id] = [
             {"id": tree.id, "name": tree.name, "contextualTitle": tree.contextual_title}
-            for tree in tree_view_container.tree_views
+            for tree in filter(
```
[the remark this answers is not in the document]
Comment (mmokko) [reply robocorp-code/codegen/views.py:250]: Yeah, haven't done much python coding in recent years, thanks good catch 👍

File: robocorp-code/codegen/commands.py [hunk 4]
```diff
@@ -387,23 +387,23 @@ def __init__(
     Command(
         "robocorp.newWorkItemInWorkItemsView",
         "New Work Item",
-        add_to_package_json=True,
+        add_to_package_json=False,
         server_handled=False,
         icon="$(new-file)",
-        hide_from_command_palette=False,
+        hide_from_command_palette=True,
```
[the remark this answers is not in the document]
Comment (mmokko) [reply robocorp-code/codegen/commands.py]: Good point, fixed.

File: robocorp-code/vscode-client/src/extension.ts (hunk 1 above)
Comment (mmokko) [reply robocorp-code/vscode-client/src/extension.ts:486]: Commented out the line.

**Pull request merged**